src/queries/fragments/image.js

```diff
@@ -0,0 +1,9 @@
+const ImageFragment = `
+    fragment ImageFragment on MediaItem {
+        uri
+        title
+        srcSet
+        sourceUrl
```

Comment:
Can we also have altTxt?

Reply:
Of course, it's provided as altText, i will update it